components/developer/gdb/manifests/sample-manifest.p5m

```diff
@@ -38,24 +39,24 @@ file path=usr/lib/$(MACH64)/libopcodes.a
 file path=usr/share/gdb/python/gdb/FrameDecorator.py
 file path=usr/share/gdb/python/gdb/FrameIterator.py
 file path=usr/share/gdb/python/gdb/__init__.py
-file path=usr/share/gdb/python/gdb/__pycache__/FrameDecorator.cpython-37.pyc
-file path=usr/share/gdb/python/gdb/__pycache__/FrameIterator.cpython-37.pyc
-file path=usr/share/gdb/python/gdb/__pycache__/__init__.cpython-37.pyc
-file path=usr/share/gdb/python/gdb/__pycache__/frames.cpython-37.pyc
-file path=usr/share/gdb/python/gdb/__pycache__/printing.cpython-37.pyc
-file path=usr/share/gdb/python/gdb/__pycache__/prompt.cpython-37.pyc
-file path=usr/share/gdb/python/gdb/__pycache__/types.cpython-37.pyc
-file path=usr/share/gdb/python/gdb/__pycache__/unwinder.cpython-37.pyc
-file path=usr/share/gdb/python/gdb/__pycache__/xmethod.cpython-37.pyc
+file path=usr/share/gdb/python/gdb/__pycache__/FrameDecorator.cpython-39.pyc
```

Comment:
We now have simple way how to filter out things like these from `sample-manifest.p5m`.  Please look at `components/python/testpath/Makefile` for an example.

Reply:
Added thanks for the hint.